This is a temporary hack around bitmap font breakage caused by
https://gitlab.freedesktop.org/freetype/freetype/-/issues/1076
also see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=997674
In a couple of years this can be removed and the small bitmap fonts will work
fine again, too.
--- a/GameDefinition/config.ui.default
+++ b/GameDefinition/config.ui.default
@@ -33,7 +33,13 @@
 ;sdlFontFile = "16x16xw.woff"
 scalableFontSize = 16
 sdlScalableSizeAdd = 0
-sdlFontFile = "16x16xw.bdf"
+; This is a temporary hack around bitmap font breakage caused by
+; https://gitlab.freedesktop.org/freetype/freetype/-/issues/1076
+; see also https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=997674
+; In a couple of years this can be removed and the small bitmap fonts will work
+; fine again, too.
+;sdlFontFile = "16x16xw.bdf"
+sdlFontFile = "16x16xw.woff"
 sdlBitmapSizeAdd = 0
 ;sdlFontFile = "8x8xb.fnt"
 ;sdlFontFile = "8x8x.fnt"
